src/components/Comments/CommentCard.tsx

```diff
@@ -100,16 +100,18 @@ const CommentCard = ({
                             rows={5}
                             id="NewPunchComment"
                             onChange={handleCommentChange}
-                        />
-                        <Button onClick={buttonClick}>Add comment</Button>{' '}
+                        />{' '}
                         {loadingStatus === AsyncStatus.LOADING ? (
-                            <Progress.Circular size={16} />
+                            <Button disabled={true}>
```

Comment:
Could you make the buttons the same width? It looks a bit odd when the button suddenly becomes smaller while loading.